sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Release History
 
-## 1.4.0b8 (Unreleased)
+## 1.4.0 (2020-08-10)
+### Added
```

Comment:
Should we use "Features" here?

Reply:
🤷‍♂️ the example in the [guidelines](https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance) shows "Added".